WordPress/wordpress-playground
# Skip Blueprint execution in the browser when persistent storage is used

Skips the Blueprint execution when WordPress is loaded from an external source. Running the same Blueprint twice installing the same plugins, overwriting the files etc. would just lead to corrupting the WordPress instance.

 ## Testing instructions

1. Go to this [local Playground URL](http://localhost:5400/website-server/?storage=browser#{%22login%22:true,%20%22plugins%22:[%22classic-editor%22],%20%22landingPage%22:%22/wp-admin/post-new.php%22}).
2. Confirm it loads the block editor
3. Refresh the page
4. Confirm it loaded without installing WordPress and that it still worked and there were no conflicts arount the classic-editor plugin
